tests/subsys/net/openthread/rpc/client/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+tests:
+  net.openthread.rpc.client:
+    sysbuild: true
+    platform_allow: native_posix
+    tags: openthread ci_build sysbuild
+    integration_platforms:
+      - native_posix
```

Comment:
I guess `native_posix` deprecated name. Please use `native_sim`. https://docs.zephyrproject.org/latest/boards/native/native_posix/doc/index.html

Reply:
Right, I forgot to update this.